exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/AttributeKeyValue.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.internal.otlp;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.KeyValue;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Key-value pair of {@link AttributeKey} key and its corresponding value.
+ *
+ * <p>Conceptually if {@link Attributes} is a Map, then this is a Map.Entry. Note that whilst {@link
+ * KeyValue} is similar, this class holds type information on the Key rather than the value.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
```

Comment:
```suggestion
 * <p>NOTE: This class is only used in the profiling signal, and exists in this module and package because its a common dependency of the modules that use it. Long term, it probably makes more sense to live in {@code opentelemetry-sdk-profiles} once such a module exists. 
 *
 * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
```